jupyter-notebooks/analysis-ready-data/ard_2_use_case_1.ipynb

```diff
@@ -81,15 +109,32 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "planet, version 2.0a6\r\n"
+      "Usage: planet [OPTIONS] COMMAND [ARGS]...\n",
+      "\n",
+      "  Planet SDK for Python CLI\n",
+      "\n",
+      "Options:\n",
+      "  --quiet           Disable ANSI control output.\n",
+      "  --version         Show the version and exit.\n",
+      "  --verbosity TEXT  Optional: set verbosity level to warning, info, or debug.\n",
+      "                    Defaults to warning.\n",
+      "  --help            Show this message and exit.\n",
+      "\n",
+      "Commands:\n",
+      "  auth           Commands for working with Planet authentication\n",
+      "  collect        Collect a sequence of JSON descriptions into a single...\n",
+      "  data           Commands for interacting with the Data API\n",
+      "  orders         Commands for interacting with the Orders API\n",
+      "  subscriptions  Commands for interacting with the Subscriptions API\n"
      ]
     }
    ],
    "source": [
-    "!planet --version"
+    "!planet - -version"
```

Comment:
we don't need this line since we're using the Client instead of the CLI in this notebook.